app/forms/support_interface/editable_until_form.rb

```diff
@@ -0,0 +1,50 @@
+module SupportInterface
+  class EditableUntilForm
+    include ActiveModel::Model
+    attr_accessor :application_form, :audit_comment, :audit_comment_description
+    attr_writer :sections, :editable_until
+
+    validates :audit_comment, presence: true
+    validates_with ZendeskUrlValidator
+
+    def non_editable_sections
+      CandidateInterface::Section.non_editable
```

Comment:
Section will need to be moved to a different namespace. We shouldn't be calling this in the `ProviderInterface`.